README.md

```diff
@@ -1317,6 +1317,9 @@ Other Style Guides
     numbers.forEach(num => sum += num);
     sum === 15;
 
+    const sum = numbers.map(num => num + 1);
```

Comment:
Sorry! I was working off the example in the issue. I should have paid more attention the sum itself. The idea was that `forEach` was in some ways suboptimal because it had side effects. The example provided was:

```
// good
const nums = [1, 2, 3];
const incs = nums.map(x => x + 1);
```

However, in retrospect, if it is to match the previous examples exactly, `map` wouldn't work in this case and reduce is the optimal solution. I see why `forEach` was used. Is the example above useful?